stories/utils/wrap.tsx

```diff
@@ -26,3 +26,11 @@ export function wrap(
         </IntlWrapper>
     );
 }
+
+export function screenshotTarget(component: JSX.Element, width = 400, height = 400) {
```

Comment:
what is this? Is it re-usable?

Reply:
Yep, plan is to have this reusable, as many stories repeat this code many times inside. I wanted to use this in place of the repeated code but it is probably not good idea to pollute this pull request with it.